src/Sylius/Bundle/CoreBundle/Command/InstallDatabaseCommand.php

```diff
@@ -54,6 +58,10 @@ protected function execute(InputInterface $input, OutputInterface $output): void
         $this->runCommands($commands, $output);
         $outputStyle->newLine();
 
-        $this->commandExecutor->runCommand('sylius:install:sample-data', [], $output);
+        $parameters = [];
+        if (null !== $suite) {
```

Comment:
Do we need to check this? If we pass just `['--fixture-suite' => $suite]` even if suite is null it should still work?

Reply:
Yes, we need it